vmdb/spec/views/host/_form.html.haml_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+include ApplicationHelper
```

Comment:
@jzigmund do we need to include ApplicationHelper?

I think set_controller_for_view is included from the view_spec_helper:

https://github.com/ManageIQ/manageiq/blob/fd0d3925b359f79f7a75b5c681583c8694f919c6/vmdb/spec/support/view_spec_helper.rb#L21-L23

Which is loaded for view from here:
https://github.com/ManageIQ/manageiq/blob/fd0d3925b359f79f7a75b5c681583c8694f919c6/vmdb/spec/spec_helper.rb#L64

Do we need it for something else?